routingv8/request.go

```diff
@@ -46,6 +46,12 @@ type CalculateMatrixRequest struct {
 	Body *CalculateMatrixBody
 }
 
+type RoutesRequest struct {
+	Origin        GeoWaypoint `json:"origin"`
```

Comment:
Since we don't use this request struct as a body in the actual request to HERE, I don't think we need the json:"xyz" annotations.